### DISCUSSION TITLE
Some issues

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#96] [General]
Hello @larspalo, by adding SequencerLabel and GeneralLabel to my panel, I realized that by closing GOodf (Last release) then reopening my file, these two elements were found "GUI label" in the list of elements of my panel. Impossible to distinguish which is Sequencer and which is General.. 

Also, when you add elements in a panel, they are automatically stacked, it is up to the user to arrange them as they wish. But in this case if you want to slide them to separate them, it's impossible, the whole stack comes together. To separate them it is necessary to note numerically a position X or Y ..

**Comment by larspalo** [comment]
Yes, this could absolutely be considered bugs to fix...

**Comment by larspalo** [comment]
Until a new release has been made you can use the test builds at https://github.com/GrandOrgue/GoOdf/actions/runs/9031189729 which should have both issues fixed.
